.changeset/mean-moons-check.md

```diff
@@ -0,0 +1,5 @@
+---
+'playroom': minor
```

Comment:
```suggestion
'playroom': patch
```

Because we're still on a 0.x version, the caret range `^0.x` won't get the favicon without a manual version bump.